status-reporter/status_reporter/status_reporter.py

```diff
@@ -95,35 +98,43 @@ def __init__(self, sampler, node_type):
             raise InitializationError('Failed initialization of status '
                                       'reporter due to:\n {issues}'.
                                       format(issues='\n'.join(issues)))
+        self._node_id = self._config.get('node_id', None)
```

Comment:
Don't we want to add the `_node_id` to the required check?

Reply:
The field is written in the install process

Reply:
Like managers_ips, cloudify_user_name and token, no?

Reply:
Well I will remove the 'cloudify_user_name' check after we will write that field automaticaly on install